### PR TITLE
Update the UPGRADE.md file

### PR DESCRIPTION
**Pull request by leofeyer**

### CODE REVIEW

**Review by ausi — DISMISSED** [review]

File: UPGRADE.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # API changes
 
+## Version 4.* to 4.10
+
+### Article CSS classes
+
+The CSS classes "first" and "last" are no longer applied to articles. Use CSS
```
Comment (ausi) [inline UPGRADE.md]: But aren’t content elements affected too?

**Comment by leofeyer** [comment]
You are right, we have also removed the classes from CEs in #2226. Did we want to do that?

File: UPGRADE.md (hunk 1 above)
Comment (leofeyer) [reply UPGRADE.md]: You are right, we have also removed the classes from CEs in #2226. Did we want to do that?

Comment (ausi) [reply UPGRADE.md]: I think so, yes.

Comment (leofeyer) [reply UPGRADE.md]: Fixed in 52641c33acc0f1389999b46164c2b53a7d360fd6 then.

**Review by ausi — APPROVED** [review]

**Pull request merged**